release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -482,6 +482,11 @@ and no other Reports or ReportQueries depend on the expiring Report, the Meterin
 removes the Report from your cluster at the end of its retention period. For more information,
 see metering Reports xref:../metering/reports/metering-about-reports.adoc#metering-expiration_metering-about-reports[expiration].
 
+[id="ocp-4-6-samples-operator"]
+==== Samples Operator on Power and Z
+
+The Samples Operator is now installed on Power and Z, and delivers Power and Z samples including images, imagestreams, and templates.
```

Comment:
Technically the Samples Operator exists on Power and Z, it just bootstraps as `Removed`. 
```suggestion
Imagestreams and templates for Power and Z architectures are now available and installed by the Samples Operator by default..
```

Reply:
Thank you! Updating now.